### ISSUE TITLE
[Accessibility]: Insufficient color contrast on monaco editor and related SaveToolbar

### ISSUE THREAD

**Issue by kopancek** [sourcegraph/sourcegraph#35716]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34120

### Problem description

<img width="474" alt="Screenshot 2022-05-19 at 17 40 55" src="https://user-images.githubusercontent.com/9974711/169339650-9ba6ee58-006d-4f5a-8959-9cee40d1fed2.png">

When using monaco editor the colors defined do not have sufficient contrast. The problem manifests both in light and dark themes.

<img width="298" alt="Screenshot 2022-05-19 at 17 42 17" src="https://user-images.githubusercontent.com/9974711/169340140-cc8b06ea-3e4e-4489-91d1-1f7c4752a833.png">

Same applies to the buttons Save / Discard changes above. If the save button is active, it does not have good contrast. If the Discard button is disabled, it does not have good contrast.

https://github.com/sourcegraph/sourcegraph/issues/34120

### Expected behavior

Buttons have good contrast.

Colors used in the monaco editor have good contrast. Since those are defined in the source code itself, it might be needed to check with a designer on which colors should actually be used with both light and dark themes.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.
